### DISCUSSION TITLE
Game Doesn't Recognize GBA rom

### DISCUSSION THREAD

**Discussion by Snowball750** [rafaelvcaetano/melonDS-android#1178] [Support]
I Beat Pokémon Heartgold yesterday And i Wanna migrate my Pokémon,But in the Game Menu There is no Migrate Button, even though I have loaded the gba rom and the save file.

Can anyone help me with this? Because I saw a comment from a guy who managed to do this and managed to migrate

I also have another question. Is this a problem with the game or with MelonDs?

Photos Below⬇️⬇️

![Screenshot_20230703-132743~2](https://github.com/rafaelvcaetano/melonDS-android/assets/138420391/5bd915a4-2f02-4ae1-aac9-adad7dc52176)
![Screenshot_20230703-132949~2](https://github.com/rafaelvcaetano/melonDS-android/assets/138420391/5ecb1571-1816-4ee9-91a7-6a510ba938f2)
(I'm Playing My GBA Games on the Myboy Emulator, which creates a folder where the game save file is.Also My Melon Ds is in Portuguese, but the buttons are the same)

**Comment by rafaelvcaetano** [comment]
Are you using a custom BIOS? Although it's not specified anywhere, a custom BIOS and firmware is required.

**Comment by Snowball750** [comment]
Thank You!I Sucefully Installed The DS Bios And Firmware  And now i Can migrate!
